api/v1alpha1/k6_types.go

```diff
@@ -27,6 +27,7 @@ type K6Spec struct {
 	Arguments   string                 `json:"arguments,omitempty"`
 	Image       string                 `json:"image,omitempty"`
 	Ports       []corev1.ContainerPort `json:"ports,omitempty"`
+	VolumeClaim string                 `json:"volumeClaim,omitempty"`
```

Comment:
See line 24. If it's now possible to pick between using a volume and a config map, the script prop should be made optional as well.